analyse_acuracy.py

```diff
@@ -0,0 +1,165 @@
+from ast import literal_eval
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+
+
+def parse_table(path):
+    pass
+
+
+
+
+def add_count_data(df, col, frames_col='frames'): 
+    # because frames is what the col t-min - t-max col is called
+    if isinstance(df.loc[0, col], str):
+        # this and the following isinstance lines are a tad fragile
+        # will address this at a later date
+        lens_p = [len(list(set(literal_eval(l)))) \
+                  for l in df[col].values]
+    else:
+        m = 'Incorrect type for counting false positives'
+        assert isinstance(df.loc[0, col], list), m
+        lens_p = [len(list(set(l))) \
+                  for l in df[col].values]
+    df[col + '_count'] = lens_p
+    df[col + '_per_frame'] = df[col + '_count'] / df[frames_col]
+    return df
+
+
+def generate_summary_df(dfs):
+    col_names = [df.columns.values for df in dfs]
+    #summary_cols = np.concatenate(col_names)
+    means = [df.mean() for df in dfs]
+    summary_cols = np.unique(np.concatenate([m.index.values for m in means]))
+    sems = [df.sem() for df in dfs]
+    rows = range(len(means))
+    summary_df = {}
+    for col in summary_cols:
+        m_values = []
+        s_values = []
+        for i in range(len(rows)):
+            if col in col_names[i]:
+                new_m = means[i][col]
+                new_s = sems[i][col]
+            else:
+                new_m, new_s = np.NaN, np.NaN
+            m_values.append(new_m)
+            s_values.append(new_s)
+        summary_df[col + '_mean'] = m_values
+        summary_df[col + '_sem'] = s_values
+    summary_df = pd.DataFrame(summary_df)
+    return summary_df
+
+
+
+
+
+
+if __name__ == "__main__":
+    from _parser import custom_parser, get_paths, track_view_base
+    # -------------
+    # Btrack tracks
+    # -------------
```

Comment:
Can you split this into a few functions, and then bottle it all into a `main()` function? Then you can just do:

```python
if __name__ == '__main__':
    main()
```

This is useful within bigger packages or even to provide an entry point, e.g. `pip install pia-tracking` will then give you access to `track-accuracy` as a command in the terminal. This is e.g. how `pip install napari` gives you `napari` as a command.